### DISCUSSION TITLE
Turbrorepo not working with a package built with webpack

### DISCUSSION THREAD

**Discussion by jaslloyd** [vercel/turborepo#1357] [Help] [answered]
Hey everybody,

I took the example design system repo and I added a new package to that instead of being build with tsup it was being built with webpack which can be seen [here](https://github.com/jaslloyd/turbo-webpack-issue) '@acme/test`. This is a small sample of a bigger project that needs to keep using webpack...

I have a package.json in the acme-test folder that has this:

```json
  "name": "@acme/test",
  "version": "0.0.0",
  "main": "./dist/index.js",
  "scripts": {
    "build": "webpack build --mode=production",
    "dev": "webpack --mode=development --watch",
    "lint": "TIMING=1 eslint src/**/*.ts* --fix",
    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
  },
```

A sample of my [webpack file](https://github.com/jaslloyd/turbo-webpack-issue/blob/main/packages/acme-test/webpack.config.js) looks like this:

```js
module.exports = {
  entry: "./src/index.jsx",
  target: "browserslist",
  output: {
    filename: "index.js",
    path: resolve("dist"),
  },
....
```

My entry file `src/index.jsx` looks like this:

```jsx
export const P = ({ children }) => <p>{children}</p>;
```

In the storybook package I am using it like so:

```
import { P } from "@acme/test";
```

When i run `npm run dev` I get the following:

`Uncaught SyntaxError: The requested module '/@fs/.../design-system-turbo/packages/acme-test/dist/index.js' does not provide an export named 'P' (at button.stories.mdx.js:7:10)`

Does anybody know if I am missing something? or configuring something wrong? It seems like it doesn't see any exports. I have also tried this with a Next.js example and ran into the same problems.

**Comment by carlos-algms** [comment, marked as answer]
same here. 
Webpack seems to be lost with the linked packages inside `node_modules`. 

There is this workaround, by setting:
``` 
// wepback.config.js 
resolve: {
    symlinks: false
}
```

It works properly in the mono-repo, I can't confirm if `yarn link` or `npm link` are going to work after that tho.
